Extension/src/LanguageServer/configurations.ts

```diff
@@ -630,8 +630,7 @@ export class CppProperties {
 
             // Replace all \<escape character> with \\<character>.
             // Otherwise, the JSON.parse result will have the \<escape character> missing.
-            readResults = readResults.replace(/\\/g, '\\\\');
-            readResults = readResults.replace(/\\\\"/g, '\\"'); // Need to revert the change to \".
+            readResults = readResults.replace(/([^\\])\\([^\"\\])/g, '$1\\\\$2'); // Exclude \" and \\.
```

Comment:
nit: An explanation of the complicated regex.

Reply:
There's an expalanation at the top and the side. Do you want me to re-organize the comment or add more?

Reply:
I guess its fine.